src/cards/ha-persistent_notification-card.html

```diff
@@ -37,12 +37,51 @@
     stateObj: {
       type: Object,
     },
+
+    scriptLoaded: {
+      type: Boolean,
+      value: false,
+    },
   },
 
+  observers: [
+    'computeContent(stateObj, scriptLoaded)',
+  ],
+
   computeTitle: function (stateObj) {
     return stateObj.attributes.title || stateObj.entityDisplay;
   },
 
+  loadScript: function () {
+    var success = function () {
+      this.scriptLoaded = true;
+    }.bind(this);
+
+    var error = function () {
+      console.error('Micromarkdown was not loaded.');
+    };
+
+    this.importHref(
+      '/static/micromarkdown-js.html',
+      success, error
+    );
+  },
+
+  computeContent: function (stateObj, scriptLoaded) {
+    var el = '';
+    var message = '';
+    if (scriptLoaded) {
+      el = this.$.pnContent;
+      message = micromarkdown.parse(stateObj.state);
```

Comment:
@justweb1 Can we try to pass a second parameter of 'True' here? It might fix relative links https://github.com/SimonWaldherr/micromarkdown.js/issues/29